templates/errors/401.html

```diff
@@ -13,13 +13,13 @@ <h1>{{ _("Sorry, you need to sign in again") }}</h1>
   {% if survey_type and (survey_type in SURVEY_TYPES_BUSINESS or survey_type in SURVEY_TYPES_DEFAULT) %}
     <p>{{ _("You will need to <a href='{url}'>sign back in</a> to access your account").format(url = business_logout_url) }}</p>
 
-  {% elif survey_type and survey_type in SURVEY_TYPES_SOCIAL %}
+  {% elif survey_type and survey_type in SURVEY_TYPES_SOCIAL or survey_type in SURVEY_TYPES_HEALTH %}
     <p>{{ _("To access this page you need to <a href='{url}'>re-enter your access code</a>.").format(url = other_logout_url) }}</p>
 
   {% else %}
-    <h3>Business surveys</h3>
+    <h2>Business surveys</h2>
     <p>{{ _("If you are completing a business survey, you need to sign back in to <a href='{url}'>your account</a>.").format(url = business_logout_url) }}</p>
-    <h3>All other surveys</h3>
+    <h2>All other surveys</h2>
```

Comment:
Existing issues, but this text even in the other files needs to be marked for translation.